### ISSUE TITLE
This line may cause mask tensor change to type LongTensor

### ISSUE THREAD

**Issue by lixinsu** [allenai/allennlp#4546]
https://github.com/allenai/allennlp/blob/f639336a2c721a2efc575397c009c958f02d9d91/allennlp/modules/token_embedders/pretrained_transformer_embedder.py#L220

This line causes the null mask tensor change to type long, and error raises when calling torch.stack(tensor_list) during batching.
